openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/http;
+
+# + clientEp - Connector http endpoint
+public client class Client {
+    http:Client clientEp;
+    # Client initialization.
+    #
+    # + clientConfig - Client configuration details
+    # + serviceUrl - Connector server URL
+    # + return - Returns error at failure of client initialization
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "localhost:9090/payloadV") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+    }
+    #
+    # + 'version - Version Id
+    # + versionName - Version Name
+    # + return - Ok
+    remote isolated function operationId04(int 'version, string versionName) returns string|error {
+        string  path = string `/v1/${'version}/v2/${'versionName}`;
```

Comment:
Check formatting.
There should be only one space between `string` and `path`

Reply:
Please note that these are only the sample files used only for test scenarios where the formatting is anyway removed to to compare the generated ballerina code and expected ballerina code.